src/main/prometheus/decc.ts

```diff
@@ -0,0 +1,83 @@
+import { PrometheusProvider, PrometheusQueryOpts, PrometheusQuery, PrometheusService } from "./provider-registry";
+import { CoreV1Api } from "@kubernetes/client-node";
+import logger from "../logger"
+
+export class PrometheusDECC implements PrometheusProvider {
+  id = "decc"
+  name = "Docker Enterprise Container Cloud"
```

Comment:
This could be more generally PrometheusStacklight?

Reply:
Good point. Much better. I will change it.